test/integration/targets/user/tasks/test_create_user_home.yml

```diff
@@ -152,3 +152,64 @@
         name: withskeleton
         state: absent
         remove: yes
+
+- name: Create user home directory with skel that contains symlinks
+  when: ansible_system == 'Linux'
+  tags: symlink_home
+  vars:
+    flag: /etc/resolv.conf
+  block:
+    - name: make tempdir for skel
+      tempfile: state=directory
+      register: tempdir
+
+    - name: copy skell to target
+      copy:
+        dest: '{{tempdir.path}}'
+        src: files/skel/
+      register: skel
+
+    - name: create the bad symlink
+      file:
+        src: '{{flag}}'
+        dest: '{{tempdir.path}}/skel/should_not_change_own'
+        state: link
```

Comment:
```console
01:24 fatal: [testhost]: FAILED! => {
01:24     "changed": false,
01:24     "invocation": {
01:24         "module_args": {
01:24             "access_time_format": "%Y%m%d%H%M.%S",
01:24             "attributes": null,
01:24             "dest": "/tmp/ansible.0w7loib5/skel/should_not_change_own",
01:24             "follow": true,
01:24             "force": false,
01:24             "group": null,
01:24             "mode": null,
01:24             "modification_time": null,
01:24             "modification_time_format": "%Y%m%d%H%M.%S",
01:24             "owner": null,
01:24             "path": "/tmp/ansible.0w7loib5/skel/should_not_change_own",
01:24             "recurse": false,
01:24             "selevel": null,
01:24             "serole": null,
01:24             "setype": null,
01:24             "seuser": null,
01:24             "src": "/etc/resolv.conf",
01:24             "state": "link",
01:24             "unsafe_writes": false
01:24         }
01:24     },
01:24     "msg": "Error while linking: [Errno 2] No such file or directory: b'/etc/resolv.conf' -> b'/tmp/ansible.0w7loib5/skel/should_not_change_own'",
01:24     "path": "/tmp/ansible.0w7loib5/skel/should_not_change_own"
01:24 }
```

Reply:
yes, seems 'expected files' are abset, that was my 3rd try, just need to create the file myself